src/test/jtx/impl/WSClient.cpp

```diff
@@ -205,6 +205,121 @@ class WSClientImpl : public WSClient
         return {};
     }
 
+    Json::Value
+    invoke(Json::Value const& cmd) override
+    {
+        using boost::asio::buffer;
+        using namespace std::chrono_literals;
+        Json::Value request;
+        if (cmd.isArray())
+        {
+            auto const num_requests = cmd.size();
+            for (unsigned i = 0; i < num_requests; ++i)
+            {
+                Json::Value const& cmdi = cmd[i];
+
+                if (cmdi.isMember(jss::params))
```

Comment:
Our existing code demands this internally and that is spread around a lot of places.  In the future I would like to see us migrate away from demanding params be an array because that is in contradiction with the json 2.0 spec (http://www.jsonrpc.org/specification section 4.2).